conference-finder/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "react-scripts": "3.2.0"
+    "react-scripts": "3.2.0",
+    "styled-components": "^5.0.0-beta.9"
```

Comment:
I see you installed 'styled-components' are you going to try and implement them in this PR?

Reply:
It's fine for now but I would recommend in future not adding them in to the PR until you've started using them :)